trimmer_decorator.rb

```diff
@@ -0,0 +1,14 @@
+require './nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  attr_accessor :nameable
+
+  def correct_name
+    @nameable.correct_name
+  end
```

Comment:
- [ ] Great job so far 👍 Kindly implement the `TrimmerDecorator` class which should check the length of a name and trim it if it is greater than 10. You've instead implemented the decorator class twice. Kindly rectify this issue 👍